addons/account/i18n/account.pot

```diff
@@ -15896,12 +15858,12 @@ msgstr ""
 
 #. module: account
 #. odoo-python
-#: code:addons/account/models/account_account.py:0
+#: code:addons/account/models/company.py:0
 #, python-format
 msgid ""
 "You cannot import the \"openning_balance\" if the opening move (%s) is "
-"already posted.                         If you are absolutely sure you want "
-"to modify the opening balance of your accounts, reset the move to draft."
+"already posted.                 If you are absolutely sure you want to "
+"modify the opening balance of your accounts, reset the move to draft."
```

Comment:
I know it's not due to your PR, but I imagine an other broke the translation without good reason, which would be so bad.
Could/Should we fix it to have the same translations?

Reply:
Maybe it's on purpose, but does not seem like it

Reply:
There's even a typo in that original string, I see: "openning_balance".
It used to be in a different file (`account_account.py`) before and the indentation changed apparently.

We can't fix .po files in the code, since they are overwritten by Transifex each week. We'd have to wait until the new .pot file is synced to Transifex, so translators can translate this new string and then the next week it would be pushed in the code by the Transifex script.

It must be broken already, since the string doesn't match exactly. I could fix the string as well, though it has nothing to do with my PR of course 🙈

Reply:
Yeah, if it's too late, it's too late, no worries :/